src/components/common/unlock-button/unlock-button.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, h, Prop } from '@stencil/core';
+import { getIsExtensionAvailable, getIsMetaMaskAvailable } from 'components/visual/unlock-panel/helpers';
+import { ProviderTypeEnum } from 'types/provider.types';
+
+@Component({
+  tag: 'mvx-unlock-button',
+  styleUrl: 'unlock-button.scss',
+})
+export class UnlockButton {
+  @Prop() buttonLabel: string;
+  @Prop() buttonIcon: HTMLElement;
```

Comment:
if you want you can go with label, icon, type.. since we know it's on the button